### ISSUE TITLE
Can no longer import VictoryChart, VictoryAxis, etc after yesterdays changes to those libraries v35.11.0

### ISSUE THREAD

**Issue by rholic-openpath** [FormidableLabs/victory-native#674]
# Bugs and Questions

### Checklist

* [ ] This is a `victory-native` specific issue. (Issues that _also_ appear in `victory` should be opened [here](https://github.com/FormidableLabs/victory/issues/new))

* [ ] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

* [ ] I am using the latest version of `victory-native`

* [ ] I have checked to make sure that my versions of `react-native` and `react-native-svg` are compatible with this version of `victory-native`. [Read about version requirements](https://github.com/FormidableLabs/victory-native/blob/main/README.md#peer-dependencies-and-version-requirements)

* [ ] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

Describe the issue you're seeing, and what you expect the behavior to be.

### Reproduction

Please check that your issue is specific to `victory-native`. If you're unsure, a good way to check is by trying your code in a sandbox. You can [fork this sandbox](https://codesandbox.io/s/m3xo745x2x) to get started. Issues that are not specific to `victory-native` should be reported [here](https://github.com/FormidableLabs/victory/issues/new).

Please list steps to reproduce your issue. Include a code sample and exact version numbers for `victory-native`, `react-native` and `react-native-svg`.

# Feature Requests

### Checklist

* [ ] My feature request is specific to `victory-native`. (Feature requests that do not _specifically_ address native functionality should be opened [here](https://github.com/FormidableLabs/victory/issues/new))

* [ ] I've read through the [Docs](https://formidable.com/open-source/victory/docs) and [Guides](https://formidable.com/open-source/victory/guides) to make sure this functionality doesn't already exist

* [ ] I've searched open issues to make sure I'm not opening a duplicate issue

### Description

Please describe the feature you're requesting in detail.
Since yesterdays push to victory-chart, victory-axis, etc. I can no loner import those using victory-native.

I get this error:
Super expression must either be null or a function

I checked the code change in v35.11.0 and it was definately around refactoring to functions.

https://github.com/FormidableLabs/victory/commit/f3d359758a80eb7994f16a85aa5a49fa95004e72#diff-b28b1ed9821f38c64524ea13e19264bb0772f8376afffd5d8a1f8277621833eb

What can be done to fix this? Is this the right place?

My imports are simple and follow the example:
import { VictoryBar, VictoryChart, VictoryTheme, VictoryStack, VictoryAxis, VictoryLabel } from "victory-native";

This guy has the same issue: https://stackoverflow.com/questions/68913710/getting-super-expression-must-either-be-null-or-a-function-in-react-native-app

**Comment by skychx** [comment]
https://github.com/FormidableLabs/victory/compare/v35.11.0...main

@boygirl

**Comment by ab-app** [comment]
+1

**Comment by chenzhe1007** [comment]
+1

**Comment by ryan-roemer** [comment]
Yep, the signatures have changed upstream, e.g. https://github.com/FormidableLabs/victory-native/blob/main/lib/index.js#L7

We'll need to review all the of default export to individual exports we did in victory.

/cc @boygirl @beccanelson

**Comment by udkhatri** [comment]
Having the same issue, and this issue has broke my app, which I intended to launch this week. please solve this thing because now replacing library is also not a great option as we have victory-native graphs at many places

**Comment by chenzhe1007** [comment]
Hi, udkhatri the solution in the stackoverflow page with the resolutions to 35.10.0 will bypass this issue. You can add the resolutions into the package.json while the proper fix is on its way

**Comment by PhongTV97** [comment]
> Hi, udkhatri the solution in the stackoverflow page with the resolutions to 35.10.0 will bypass this issue. You can add the resolutions into the package.json while the proper fix is on its way

After I'm insatll victory-native version 35.3.3 and import { VictoryBar } from 'victory-native' then:

TypeError: Super expression must either be null or a function
[Wed Aug 25 2021 16:47:33.488]  ERROR    Invariant Violation: Module AppRegistry is not a registered callable module (calling runApplication)
[Wed Aug 25 2021 16:47:34.526]  ERROR    Invariant Violation: Module AppRegistry is not a registered callable module (calling runApplication)

Can you help me?  Thanks

**Comment by chenzhe1007** [comment]
Yes
The temporary solution is to add the ‘resolutions’ into your package.json
and pin the vitcotry version to 35.10.0

Pls refer to the stack overflow link in the original post.

Zhe

**Comment by TomCorvus** [comment]
Hi,
same problem here. Maybe a fix today https://github.com/FormidableLabs/victory/issues/1946#issuecomment-905639362

`resolutions` is a good fix but I can't using it on monorepo with AppCenter which using npm instead of yarn.
